src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/ProductType.cs

```diff
@@ -265,6 +268,36 @@ public ProductType(IMediator mediator, IDataLoaderContextAccessor dataLoader)
 
             Field<ListGraphType<OutlineType>>("outlines", resolve: context => context.Source.IndexedProduct.Outlines);//.RootAlias("__object.outlines");
 
+            Field<ListGraphType<BreadcrumbType>>("breadcrumbs", resolve: context =>
+            {
+                var breadcrumbs = new List<Breadcrumb>();
+                var store = context.GetArgumentOrValue<Store>("store");
+                var cultureName = context.GetValue<string>("cultureName");
+
+                var outline = context.Source.IndexedProduct.Outlines.FirstOrDefault(outline => outline.Items.Any(item => item.Id == store.Catalog && item.SeoObjectType == "Catalog"));
```

Comment:
Please move this code to the  extension method that will be common for products and categories

Reply:
done